scripts/test-liquiditySwap-swapAndDeposit-swapAll-WETH-DAI.js

```diff
@@ -26,42 +23,59 @@ async function main() {
   // Build a swap on ParaSwap
   const amount_to_swap = ethers.utils.parseEther('10.01');
```

Comment:
I'm a bit confused about this line.

Does this mean that I have to essentially "add a margin" on top or the getRate call when we know we want to swap max?

Reply:
Yes, if you want to swap max, the amount to swap must be at least as big as the balance, so you need to add a little margin to account for any interest that might be accrued before the transaction is mined.